rubicon/objc/api.py

```diff
@@ -667,6 +678,17 @@ def release(self):
         self._needs_release = False
         send_message(self, "release", restype=objc_id, argtypes=[])
 
+    def autorelease(self):
+        """
+        Decrements the receiver’s reference count at the end of the current autorelease pool block
+
+        The objc object is sent a dealloc message when its reference count reaches 0. If called,
+        the object will not be released when the Python object is deallocated.
+        """
+        self._needs_release = False
+        result = send_message(self, "autorelease", restype=objc_id, argtypes=[])
```

Comment:
On the other hand, this is correct and necessary to avoid a second `release` if user code calls `autorelease`. Good catch - I forgot about that before.